data/remnant/remnant ships.txt

```diff
@@ -1052,6 +1052,8 @@ ship "Penguin"
 		"outfit scan speed" 1
 		"tactical scan power" 50
 		"atmosphere scan" 100
+		"ion protection" 0.25
+		"ion resistance" 0.05
```

Comment:
If ion storms are a problem with trying to dodge the Archon then I'd prefer if the ion storm spawn rate was either reduced or removed entirely instead of adding ion protection to more Remnant ships. For reference, the current spawn rate of ion storms in Nenia is current about every 7 minutes on average.

Reply:
And as to reducing the hazards in Nenia, I want those hazards to remain full strength to maintain the threat to any other ships that might be there, such as escorts.

Maintaining the threat to other ships is more important than the rather minor tactical advantage this gives to two non-combatant ships.